### PR TITLE
Update Kubernetes versions.

### PR DESCRIPTION
**Pull request by fasaxc**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: getting-started/kubernetes/requirements.md
Comment (lwr20) [inline getting-started/kubernetes/requirements.md:20]:
```suggestion
on Kubernetes v1.15 or below.  v1.16-v1.18 may work, but they are no longer tested. 
```
Otherwise 1.18 is in limbo

**Review by lwr20 — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
